src/services/MarkdownRenderService.ts

```diff
@@ -37,15 +37,34 @@ class MarkdownRenderService {
                 }
             }
             // Array modifiers
-            if (array && SUGGESTER_REGEX.TEXT_ARRAY.test(markdownString)) {
-                let alternativeString = "";
-                markdownString
-                    .replaceAll(SUGGESTER_REGEX.TEXT_ARRAY, "$2$3$4")
-                    .split(",")
-                    .forEach((item) => {
-                        alternativeString = alternativeString.concat(`- ${item.trim()}\n`);
+            if (array && SUGGESTER_REGEX.TEXT_ARRAY.test(markdownString) && !SUGGESTER_REGEX.IS_MD_LINK.test(markdownString) && !SUGGESTER_REGEX.IS_URL_LINK.test(markdownString)) {
+                // let alternativeString = "";
```

Comment:
Just remove the commented code to do not accumulate code smells

Reply:
i removed it - i hope you can see the updated version